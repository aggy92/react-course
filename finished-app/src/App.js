import React from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import CardList from './components/CardList';
import Details from './components/Details';
import Header from './components/Header';

function App() {
  return (
    <Router>
      <Header />
      <Switch>
        <Route exact path="/">
          <CardList />
        </Route>
        <Route path="/pokemon/:id">
          <Details />
        </Route>
      </Switch>
    </Router>
  );
}

export default App;
