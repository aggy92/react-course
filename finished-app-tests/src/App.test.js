import React from "react";
import { mount } from "enzyme";
import App from "./App";

test("renders learn react link", () => {
  const wrapper = mount(<App />);
  console.log(wrapper);
  expect(wrapper.html()).toMatchSnapshot();
});

test("test", () => {
  const obj = { foo: 43 };
  expect(obj).toMatchSnapshot();
});
