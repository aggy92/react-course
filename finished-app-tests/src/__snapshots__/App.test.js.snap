// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `"<h1 class=\\"sc-AykKC gzshXR\\">Pokedex</h1><input class=\\"sc-AykKG iZHRia\\"><div class=\\"sc-AykKF nlbLF\\"></div>"`;

exports[`test 1`] = `
Object {
  "foo": 43,
}
`;
